waspc/data/Generator/templates/react-app/src/auth/pages/OAuthCodeExchange.js

```diff
@@ -0,0 +1,55 @@
+{{={= =}=}}
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+
+import config from '../../config.js'
+import api, { setAuthToken } from '../../api.js'
+
+// This component is rendered in response to an OAuth 2.0 redirect request,
+// where it takes all of the query string parameters and sends them to the Wasp
+// API server to complete the OAuth process. Upon completion, we will
+// obtain a JWT and redirect to the auth success path.
+const OAuthCodeExchange = (props) => {
+  const history = useHistory()
+  // NOTE: Different auth methods will have different Wasp API server validation paths.
+  // This helps us reuse one component for various methods (e.g., Google, Facebook, etc.).
+  const handleOauthRedirectPath = props.handleOauthRedirectPath
```

Comment:
```suggestion
  const pathToApiServerRouteHandlingOauthRedirect = props.handleOauthRedirectPath
```
otherwise it sounds like it handles the oauth redirect path